### DISCUSSION TITLE
I guess having an "israel affiliate" founder explains why "most GUI applications will not run at the moment", regardless of both Linux and Darwin being OPEN SOURCE UNIX flavor OS's. Seems obvious that the project founder simply wanted to make something that SOUNDED important and not actually do much after that (aside from claiming credit for anything the community contributed  to the project after that).

### DISCUSSION THREAD

**Discussion by 01GOD** [darlinghq/darling#1529] [General]
I guess having an "israel affiliate" founder explains why "most GUI applications will not run at the moment", regardless of both Linux and Darwin being OPEN SOURCE UNIX flavor OS's. Seems obvious that the project founder simply wanted to make something that SOUNDED important and not actually do much after that (aside from claiming credit for anything the community contributed  to the project after that).

**Comment by 01GOD** [comment]
Surprised he's not been busy scraping data from Internet forums and putting that into an ML model with a natural lang interface and calling that "AI" to scam some investors, or busy mass murdering babies in Palestine instead.

**Comment by LubosD** [comment]
Says somebody whose GH profile is full of crappy useless stuff.

Am Israel chai.

Good bye.
